test/helpers.test.ts

```diff
@@ -500,27 +513,27 @@ describe('Vision helper methods', () => {
 
       client
         .productSearch(request)
-        .then(r => {
-          const response = r[0];
-
-          assert.deepStrictEqual(response, {
-            localizedObjectAnnotations: [{dummy: 'response'}],
-          });
-
-          assert(batchAnnotate.callCount === 1);
-          assert(batchAnnotate.calledWith({requests: [request]}));
-        })
+        .then(
+          (r: [prototypes.google.cloud.vision.v1.IAnnotateImageResponse]) => {
+            const response = r[0];
+
+            assert.deepStrictEqual(response, {
+              localizedObjectAnnotations: [{dummy: 'response'}],
+            });
+
+            assert(batchAnnotate.callCount === 1);
+            assert(batchAnnotate.calledWith({requests: [request]}));
+          }
+        )
         .catch(assert.ifError);
     });
-
+    // Problem: checked the client which does not have objectLocalization.
```

Comment:
Is this comment needed?

Reply:
right, that was there as reminder for debugging, removed it, thank you!